File: Iniciante/Python/1012.Área.py
pi = 3.14159

A, B, C = input().split()

A = float(A)
B = float(B)
C = float(C)

areaTriangulo = (A*C)/2
areaCirculo = pi * pow(C, 2)
areaTrapezio = ((A+B)*C)/2
areaQuadrado = B*B
areaRetangulo = A*B

print('TRIANGULO: {:.3f}'.format(areaTriangulo))
print('CIRCULO: {:.3f}'.format(areaCirculo))
print('TRAPEZIO: {:.3f}'.format(areaTrapezio))
print('QUADRADO: {:.3f}'.format(areaQuadrado))
print('RETANGULO: {:.3f}'.format(areaRetangulo))

File: Iniciante/C++/1012.Área.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main(void){
    double A, B, C;
    double triangulo_retangulo, circulo, trapezio, quadrado, retangulo;
    double pi = 3.14159;

    cin >> A >> B >> C;

    triangulo_retangulo = (A*C)/2.0;
    circulo = pi * pow(C, 2);
    trapezio = ((A+B)*C)/2.0;
    quadrado = B*B;
    retangulo = A*B;

    cout << "TRIANGULO: " << fixed << setprecision(3) << triangulo_retangulo << endl;
    cout << "CIRCULO: " << fixed << setprecision(3) << circulo << endl;
    cout << "TRAPEZIO: " << fixed << setprecision(3) << trapezio << endl;
    cout << "QUADRADO: " << fixed << setprecision(3) << quadrado << endl;
    cout << "RETANGULO: " << fixed << setprecision(3) << retangulo << endl;

    return 0;
}